pkg/webservice/handlers.go

```diff
@@ -521,3 +523,24 @@ func updateConfiguration(conf string) (string, error) {
 	}
 	return "", fmt.Errorf("config plugin not found")
 }
+
+func getPartitionQueues(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	writeHeaders(w)
+	_, partitionExists := vars["partition"]
```

Comment:
Please save here the partition name instead of ignoring it, so in the line 536 you can directly use it

Reply:
Taken care.